nemo_megatron_launcher/requirements.txt

```diff
@@ -0,0 +1,6 @@
+hydra-core==1.1.1
```

Comment:
@yaoyu-33 is it necessary to precisely pin these version?
Most of these libs are already NeMo requirements, so can we remove them from here?

Reply:
cool. I will double check these requirements

Reply:
@okuchaiev: I think @markelsanz14 made a point here. "This is what needs to be installed in the login node of the cluster. So it might not necessarily have NeMo installed." 
On cluster login node, we don't need to install the entire nemo to use the launcher, where we only need to install the required packages in this requirements